enterprise/cmd/frontend/internal/authz/resolvers/repositories.go

```diff
@@ -35,39 +35,65 @@ type repositoryConnectionResolver struct {
 // 🚨 SECURITY: It is the caller's responsibility to ensure the current authenticated user
 // is the site admin because this method computes data from all available information in
 // the database.
+// This function takes returns a pagination of the repo IDs
+// r.ids - the full slice of sorted repo IDs
+// r.after - (optional) the repo ID to start the paging after (does not include the after ID itself)
+// r.first - the # of repo IDs to return
```

Comment:
FYI this would not result in a good-looking formatting in almost all code-intel features, e.g. in Sourcergaph

<img width="610" alt="CleanShot 2022-04-06 at 11 08 42@2x" src="https://user-images.githubusercontent.com/2946214/161888407-a65133ae-d1bd-4b05-8793-7ef784f0ead4.png">

An indentation is needed for preserving the line separators:

```suggestion
//  r.ids - the full slice of sorted repo IDs
//  r.after - (optional) the repo ID to start the paging after (does not include the after ID itself)
//  r.first - the # of repo IDs to return
```